src/App.jsx

```diff
@@ -183,8 +185,8 @@ function App() {
       variant: "info",
     });
   };
-
   return (
+    <UserContext.Provider value={user}>
```

Comment:
hii, you don't need to use user context, you can directly access/get the current user from firebase. Check other components , how they're getting current user from auth object

Reply:
this will resolve the merge conflict as well, automatically

Reply:
Okay.